### ISSUE TITLE
Quick and Dirty Dark Theme

### ISSUE THREAD

**Issue by spec1re** [fossar/selfoss#1408]
Here a quick and dirty howto dark theme:

1. Get Dark Reader extension for [Chrome/Firefox/Edge/etc.](https://darkreader.org/)
2. [Follow this guide here](https://github.com/darkreader/darkreader/issues/604#issuecomment-661107893).
3. Save the css file as user.css and upload to your selfoss root. 

Done. 

Or just take this one attached here, where the bg.gif glitch is already fixed. [user.css.zip](https://github.com/fossar/selfoss/files/10678130/user.css.zip)

**Comment by jtojnar** [comment]
Thanks. I have attempted a dark theme [in the past](https://github.com/fossar/selfoss/issues/1094#issuecomment-466178382) but I got stuck on dealing with images, see https://github.com/fossar/selfoss/issues/1376.

**Comment by spec1re** [comment]
Yes, that was the first theme I tried but it made me look for an alternative. ;)
